src/http/response.ts

```diff
@@ -126,7 +126,7 @@ export class Response {
    */
   public render(
     ...args: unknown[]
-  ): string | boolean {
+  ): Promise<boolean | string> | boolean | string {
```

Comment:
Breaking change i guess?

Anyone using render now will fall into a race condition where the response is returned in the resource method before the body is set?

Reply:
the tests right now show that it works as it did before. i can add async tests to make sure though

Reply:
i cant see any awaits inside the `render` function?

Reply:
yea there aren't any. it's to handle template engines that use `await`. for example, this has `Promise<boolean|string>` now because of Eta. Eta requires `await renderFile()`. when people use Eta, they're going to define the render method in Tengine and call `await renderFile()`, that's where Drash's `render()` method needs to return a `Promise`.

Reply:
what `Tengine` will do is replace Drash's `render()` method with Eta's `await renderFile()` method. this is the same for any other template engine if they use `await`.

Reply:
Ah yeah gotcha